docker/dev/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts
+
+# build-arg with youtube api key
+ARG YOUTUBE_API_KEY
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY . .
+RUN npm install
+
+# change the env variable
+RUN sed -i "s|API_KEY_GOES_HERE|$YOUTUBE_API_KEY|g" /usr/src/app/env/development.env
```

Comment:
It doesn't seem like a good idea to do a replacement here. I think it would be better to just overwrite the file in the correct format.